### ISSUE TITLE
configure with python3 as /usr/bin/python

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1033]
Originally reported of Google Code with ID 977.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

<b>What distribution are you using?</b>
Archlinux
ganeti-2.11/2.10 from source


<b>What steps will reproduce the problem?</b>
1. export PYTHON=/usr/bin/python2
2. ./configure [...]

<b>What is the expected output? What do you see instead?</b>
SyntaxError: Missing parentheses in call to 'print'
  File "./autotools/print-py-constants", line 46
    print "%s" % socket.AF_INET
             ^

<b>Please provide any additional information below.</b>
I'd like to change all /usr/bin/python to /usr/bin/python2 but I noticed that for e.g. debian-sqeeze doesn't have a /usr/bin/python2

```
Originally added on 2014-10-17 22:00:47 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
submited patch

as i noticed that there was a call of "python [...]" in the Makefile(.am)
```
Originally added on 2014-10-18 00:11:34 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with d7353fe039240929753fac651400f2237b18660b
```
Originally added on 2014-10-21 08:11:00 +0000 UTC.

Changed State: Fixed
